src/Pages/Layout/Isna/data/Notifications.js

```diff
@@ -0,0 +1,31 @@
+import isBefore from 'date-fns/is_before';
+
+const allNotifications = [];
+
+const notifications = isLocalStorageAvailable()
+  ? allNotifications.filter(byExpiration).filter(byStatusInStorage)
+  : [];
+
+function byExpiration(notification) {
```

Comment:
This function is doing a mutation and it's name doesn't express this and it's also used for filtering were you would expect to use a pure fn, how do you thing this could be improved?